geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseFeatureIndex.scala

```diff
@@ -45,12 +46,14 @@ object HBaseFeatureIndex extends HBaseIndexManagerType {
   val DataColumnQualifierDescriptor = new HColumnDescriptor(DataColumnQualifier)
 
   case class ScanConfig(hbaseFilters: Seq[HBaseFilter],
-                        entriesToFeatures: Iterator[Result] => Iterator[SimpleFeature])
+                        columnFamily: Array[Byte],
+                        entriesToFeatures: Iterator[Result] => Iterator[SimpleFeature],
+                        reduce: Option[(CloseableIterator[SimpleFeature]) => CloseableIterator[SimpleFeature]])
```

Comment:
it doesn't seem like `columnFamily` and `reduce` are used, this change can be reverted I think